### DISCUSSION TITLE
Respiratory

### DISCUSSION THREAD

**Discussion by segayi91** [community/community#103031] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi , community I need your help!!
Why I cant access my website it just shows me readme text I use main branch  & uploading file from my computer.

**Comment by ghost** [comment]
It can be because you index.html is not on the root folder. This criteria is for GitHub pages. This is based on the things I learned

**Reply by ghost** [reply]
And name your html file as index.html and try if you had named it any other

**Reply by ghost** [reply]
GitHub Pages will look for an index.html , index.md , or README.md file as the entry file for your site. If your publishing source is a branch and folder, the entry file must be at the top level of the source folder on the source branch.

**Reply by ghost** [reply]
Refer this Documentation

[Doc](https://gist.github.com/TylerFisher/6127328)

**Reply by ghost** [reply, marked as answer]
Hope I was helpful, 😁🙏
